### PR TITLE
Export component types

### PR DESCRIPTION
**Pull request by Danny-Robinson**
Now exports component types

### CODE REVIEW

File: src/ts/store.tsx [hunk 1]
```diff
@@ -4,6 +4,8 @@ export type ComponentType<P> =
   | React.ComponentClass<P>
   | React.StatelessComponent<P>;
 
+
+//  @internal
```
Comment (JakeSidSmith) [inline src/ts/store.tsx:8]: Must be a doc comment.

```suggestion
/**
 * @internal 
 */
```

File: src/ts/index.ts [hunk 2]
```diff
@@ -36,12 +35,60 @@ export { default as Section } from './components/content/section';
...
 export { default as Table } from './components/tables/table';
 export { default as TableBody } from './components/tables/table-body';
 export { default as TableCell } from './components/tables/table-cell';
 export { default as TableHead } from './components/tables/table-head';
 export { default as TableHeader } from './components/tables/table-header';
 export { default as TableRow } from './components/tables/table-row';
+export { default as Tabs } from './components/tabs/tabs';
 export { default as Well } from './components/misc/well';
+
+export { AlertProps } from './components/misc/alert';
```
Comment (JakeSidSmith) [inline src/ts/index.ts:48]: All of these should be able to be exported on the same line e.g.

```ts
export { default as Alert, AlertProps } from './components/misc/alert';
```

Comment (JakeSidSmith) [reply src/ts/index.ts:48]: Also, please don't reorder any of the exports.

**Review by JakeSidSmith — CHANGES_REQUESTED** [review]

File: src/ts/index.ts (hunk 2 above)
Comment (Danny-Robinson) [reply src/ts/index.ts:48]: i made them alphabetical order :(

**Review by JakeSidSmith — APPROVED** [review]
Please run `npm version patch`

**Review by JakeSidSmith — APPROVED** [review]

**Pull request merged**